bundler.d/vmware.rb

```diff
@@ -1,3 +1,5 @@
 group :vmware do
-  gem 'fog-vsphere', '>= 3.6.0', '< 4.0'
+  # Temp pin to 3.6.0 because with 3.6.1
+  # test test/controllers/api/v2/hosts_controller_test is failing
+  gem 'fog-vsphere', '3.6.0'
```

Comment:
We can also pin it with excluding 3.6.1, assuming 3.6.2 will work again:
```suggestion
  gem 'fog-vsphere', '~> 3.6', '!= 3.6.1'
```